services/minespace-web/src/components/pages/Project/InformationRequirementsTableStepForm.tsx

```diff
@@ -103,7 +114,7 @@ const StepForms = ({
                 state: { current: 2 },
               });
             }}
-            disabled={!uploadedSuccessfully && !project?.information_requirements_table?.irt_guid}
+            disabled={!uploadedSuccessfully}
           >
```

Comment:
The second part of the condition made it so that I could click the button and it would take me in circles (if I didn't upload a document).